nfc/CHANGELOG.rst

```diff
@@ -9,6 +9,11 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+nRF Connect SDK v1.8.0
+**********************
+
+No changes in this release.
```

Comment:
Notable changes
===============

* IE Termination Header is now allowed to be not present in case of incoming frames with no payload IEs and no payload. (KRKNWK-11875)

Bug fixes
=========

* Fixed an issue where the notification queue could be overflowed under stress. (KRKNWK-11606)
* Fixed an issue where `nrf_802154_transmit_failed` callout would not always propagate the frame properties correctly. (KRKNWK-11605)

Reply:
Does "notable changes" mean there are other changes we're not listing?